pkg/features/kube_features.go

```diff
@@ -390,6 +390,18 @@ const (
 	//
 	// Enables the kubelet's pod resources grpc endpoint
 	KubeletPodResources utilfeature.Feature = "KubeletPodResources"
+
+	// owner: @davidz627
+	// alpha: v1.14
+	//
+	// Enables the in-tree storage to CSI Plugin migration feature.
+	CSIMigration utilfeature.Feature = "CSIMigration"
+
+	// owner: @davidz627
+	// alpha: v1.14
+	//
+	// Enables the GCE PD in-tree driver to GCE CSI Driver migration feature.
+	CSIMigrationGCE utilfeature.Feature = "CSIMigrationGCE"
```

Comment:
Is it necessary to have a CSIMigration feature, and features for individual drivers?

Reply:
yes, since we have to progress each plugin separately based on qualification. And we also want a way to turn the whole feature off if its broken